packages/backend/src/graphql/queries/test-connection.ts

```diff
@@ -52,8 +53,15 @@ const testConnection = async (
   let isStillVerified
   try {
     isStillVerified = !!(await app.auth.isStillVerified($))
-  } catch {
+  } catch (err) {
     isStillVerified = false
+    logger.error(`Error verifying CONNECTION ID: ${params.connectionId}`, {
+      errorDetails: {
```

Comment:
nit: should we add an `event` field to make it easier to filter? I typically do stuff like this to make it easier to search on DD.

https://github.com/opengovsg/plumber/blob/cd8ccd11d77841cdb215581b03dd2dd4870be68c/packages/backend/src/controllers/webhooks/handler.ts#L63

Reply:
thanks for the reco